### ISSUE TITLE
method to load new slides

### ISSUE THREAD

**Issue by bardolf69** [megasanjay/vue3-marquee#270]
**Is your feature request related to a problem? Please describe.**
I'm using the marquee as a stock ticker and it would be nice to be able to update data being streamed without re-rendering the marquee completely.

**Describe the solution you'd like**
It would be nice to have a method that allows you to send a new set of data that replaces the current data once the current loop is finished.  To explain a little clearer, what I'm proposing is that once the data is received by the method it waits until the last piece of current content and then begins feeding the new content from that point on.  So you would have [old3] [old4] [new1] [new2].  That would make the transition seamless.

**Comment by bardolf69** [comment]
Great, I'll keep an eye out for it.  What I'm hoping to do is hook the marquee to a a websocket feed.
